### ISSUE TITLE
Pythran points to the wrong line on silly error

### ISSUE THREAD

**Issue by jeanlaroche** [serge-sans-paille/pythran#1102]
When compiling this:
```
import numpy as np

# pythran export test(float)
def test(x):
    return 2*x

# pythran export test3(float,)  # <- This is where the error is
def test3(x):
    return 2*x
```
Pythran reports 

> CRITICAL
> I am in trouble. Your input file does not seem to match Pythran's constraints...
> E: Invalid Pythran spec near ')' (line 5)

But the error is on line 7, not 5. On a much larger program this can be very confusing because the real error is nowhere near where pythran says it is.

**Comment by serge-sans-paille** [comment]
Thxs for the report! I can reproduce on my Linux box, some homework for tonight :-)

**Comment by serge-sans-paille** [comment]
Got it. The short story is: never trust a regular expression

**Comment by serge-sans-paille** [comment]
Should be fixed by #1107, thanks again for the bug report!

**Comment by jeanlaroche** [comment]
> 
> 
> Got it. The short story is: never trust a regular expression

Very untrustworthy they are!

**Comment by serge-sans-paille** [comment]
Fixed in #1107
